native/src/routes/SearchModal.tsx

```diff
@@ -54,9 +62,10 @@ const SearchModal = ({
   const contentLanguageReturn = useSearch(documents, debouncedQuery)
   const fallbackLanguageReturn = useSearch(fallbackLanguageDocuments, debouncedQuery)
   const searchResults = contentLanguageReturn.data.concat(fallbackLanguageReturn.data)
+  const limitedResults = searchResults.slice(0, MAX_SEARCH_RESULTS)
```

Comment:
🙃 `limited` sounds a little like they are of insufficient quality. Not sure how to name it better. We could also just do the following though, not sure if thats better though. Same for web.
```suggestion
  const searchResults = contentLanguageReturn.data.concat(fallbackLanguageReturn.data).slice(0, MAX_SEARCH_RESULTS)
```